tests/test_recorders.py

```diff
@@ -186,18 +189,18 @@ def test_numpy_index_parameter_recorder(simple_storage_model):
     assert(res_rec.data.shape == (5, 1))
     assert_allclose(res_rec.data, np.array([[7, 4, 1, 0, 0]]).T, atol=1e-7)
     assert (lvl_rec.data.shape == (5, 1))
-    assert_allclose(lvl_rec.data, np.array([[0, 1, 2, 2, 2]]).T, atol=1e-7)
+    assert_allclose(lvl_rec.data, np.array([[0, 0, 1, 2, 2]]).T, atol=1e-7)
```

Comment:
See comment at the top of the test. Essentially the parameter is recorded at the start of the timestep, but the storage is recorded at the end.